src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.spec.ts

```diff
@@ -20,6 +20,7 @@ import { HearingsService } from '../../../services/hearings.service';
 import { LocationsDataService } from '../../../services/locations-data.service';
 import * as fromHearingStore from '../../../store';
 import { HearingEditSummaryComponent } from './hearing-edit-summary.component';
+import { cold } from 'jasmine-marbles';
```

Comment:
Minor comment - generally external imports such as cold should be above relative internal imports (as is the case in majority of files), approving regardless